server/usr/libexec/nethserver/postfix-get-group

```diff
@@ -30,6 +30,7 @@ do
     elif [ -z "$group" ]; then
         echo 500 Group%20name%20not%20provided
     else
+	group=$(echo -n "$group" | sed -e 's/^"\(.*\)"$/\1/; s/%20/ /g')
```

Comment:
Double quotes are around the local address part.
The `"$` regexp part shouldn't match an input like `"domain%20admins"@example.com`. 

```suggestion
	group=$(echo -n "$group" | sed -e 's/"//g; s/%20/ /g')
```